doc/api/jstp.md

```diff
@@ -174,6 +174,7 @@ Execute an action.
 - `action` [`<string>`][], action name
 - `actionArgs` [`<Object>`][object]
   - `args` [`<Object>`][object]
+  - `context` [`<Object>`][object]
```

Comment:
I'd prefer to move `context` before `args` to correlate with comments in the code.

Reply:
There are no comments for the methods in `lib/remote.provider.jstp.api.js` and the point of this document is to document the arguments and behavior of the remote methods implemented there.